client-v2/src/shared/components/article/ArticleBodyDefault.tsx

```diff
@@ -27,8 +27,8 @@ import CollectionItemDraftMetaContent from '../collectionItem/CollectionItemDraf
 import ColouredQuote from '../collectionItem/CollectionItemQuote';
 
 const ThumbnailPlaceholder = styled(BasePlaceholder)`
-  width: 130px;
-  height: 67px;
+  width: 83px;
+  height: 50px;
```

Comment:
The placeholder should be the same size as image so have cut this down to the new measurements. Let me know if this hardcoding is too naughty